src/Sulu/Bundle/MediaBundle/Resources/js/components/SingleMediaDropzone/SingleMediaDropzone.js

```diff
@@ -115,49 +115,62 @@ class SingleMediaDropzone extends React.Component<Props> {
 
         return (
             <Dropzone
-                className={mediaContainerClass}
-                disableClick={uploading}
                 disabled={disabled}
                 multiple={false}
+                noClick={uploading}
                 onDragEnter={this.handleDragEnter}
                 onDragLeave={this.handleDragLeave}
                 onDrop={this.handleDrop}
             >
-                {image && !this.imageError &&
-                    <Fragment>
-                        <img className={singleMediaDropzoneStyles.thumbnail} key={image} src={image} />
-                        {this.imageLoading && <Loader />}
-                    </Fragment>
-                }
-                {(!image || this.imageError) && mimeType &&
-                    <div className={singleMediaDropzoneStyles.mimeTypeIndicator}>
-                        <MimeTypeIndicator iconSize={100} mimeType={mimeType} />
-                    </div>
-                }
-                {!image && !mimeType &&
-                    <div className={singleMediaDropzoneStyles.emptyIndicator}>
-                        <Icon name={emptyIcon} />
-                    </div>
-                }
-
-                {!uploading
-                    ? <div className={singleMediaDropzoneStyles.uploadIndicatorContainer}>
-                        <div className={singleMediaDropzoneStyles.uploadIndicator}>
-                            <div>
-                                <Icon className={singleMediaDropzoneStyles.uploadIcon} name={UPLOAD_ICON} />
-                                {uploadText &&
-                                    <div className={singleMediaDropzoneStyles.uploadInfoText}>{uploadText}</div>
+                {({getInputProps, getRootProps}) => (
+                    <Observer>
```

Comment:
Interesting didn't know that something like this exists 😲

Reply:
Yeah, and it is necessary in that case... The thing is that because the `react-dropzone` now only accepts a function, the stuff we add here is actually rendered by `react-dropzone`, and then the `SingleMediaDropzon` does not seem to observe it anymore. Therefore we need to add another anonymous `Observer` here.